src/uberjob/_run.py

```diff
@@ -191,7 +191,7 @@ def run(
     :param output: An optional symbolic output specification. It will be converted via :meth:`~uberjob.Plan.gather`.
     :param registry: An optional :class:`~uberjob.Registry` that may specify a
                      :class:`~uberjob.ValueStore` for each :class:`~uberjob.graph.Node`.
-    :param dry_run: If True, returns the plan that would be run without actually running it.
+    :param dry_run: If True, returns the physical plan and output node without actually running it.
```

Comment:
dry_run should really return both the physical plan and output node, rather than just the physical plan, so I switched that

Reply:
I agree that this looks like a previous oversight.